app/SectionApp/TodoContainer/Values/StepsOfTodoValue.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\SectionApp\TodoContainer\Values;
+
+use App\Ship\Values\BaseValue;
+
+/**
+ * GetSteps representation
+ */
+class StepsOfTodoValue extends BaseValue
```

Comment:
StepValue là được rồi 
```suggestion
/**
 * Step representation
 */
class StepValue extends BaseValue
```